blurb/blurb.py

```diff
@@ -93,7 +93,7 @@
 #.. section: IDLE
 #.. section: Tools/Demos
 #.. section: C API
-
+{section}
```

Comment:
why don't you keep the same principle with the `.. bpo: {bpo}`? at this line, you already have a constructed section.
You build the section at that line https://github.com/python/core-workflow/pull/312/files#diff-1bd652c5f766ddebec6cd6dc8f27b3b1R927

Do you make the construction before because you don't want to have a conditional template? is it right?